dashboard/app/views/layouts/_header.html.haml

```diff
@@ -10,6 +10,8 @@
 
 - show_bug_links = (script_level && script.show_report_bug_link?) || (level && level.try(:is_project_level))
 
+= render partial: 'shared/header_progress_popup'
```

Comment:
I think the main difference is that `_header` is rendered on all studio.code.org pages but `_header_progress` is only rendered on tutorial levels.